modules/core/src/main/java/org/apache/ignite/internal/processors/metric/impl/MetricUtils.java

```diff
@@ -234,4 +236,29 @@ public static Map<String, Class<?>> systemViewAttributes(SystemView<?> sysView)
 
         return attrs;
     }
+
+    /**
+     * Extract system view from systemViewManager. Both "SQL" and "Java" styles of system view name are supported.
+     *
+     * @param systemViewManager Manager to extract view from.
+     * @param viewName System view name.
+     * @return System view with viewName. Null if the view does not exist.
+     */
+    @Nullable public static SystemView<?> getSystemViewByName(GridSystemViewManager systemViewManager, String viewName) {
```

Comment:
We don't use prefixes like `get`, `set`, `is` in function names within internal packages